workflow/envs/filter_vcf.yaml

```diff
@@ -2,5 +2,3 @@ channels:
   - conda-forge
   - bioconda
   - defaults
-dependencies:
```

Comment:
do we need to add hydra-genetics as a dependency here? If a job from this rule is submitted to a cluster it could be that it will not have access to hydra. @jonca79

Reply:
Den finns ju med i requirements.txt.